metricbeat/docs/fields.asciidoc

```diff
@@ -3874,6 +3874,21 @@ The HTTP payload received
 json metricset
 
 
+[float]
+== server Fields
+
+server
+
+
+
+[float]
+=== http.server.example
```

Comment:
i have removed this field from the asciidoc. during restart/multiple definitions we would error out with a critical message saying port is already in use.